plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -175,6 +175,7 @@ func ClusterRoles() []rbac.ClusterRole {
 					// do not expand this pattern for openapi discovery docs
 					// move to a single openapi endpoint that takes accept/accept-encoding headers
 					"/swagger.json", "/swagger-2.0.0.pb-v1",
+					"/openapi", "/openapi/*",
```

Comment:
Saving the place for `/openapi/v3` in future. Also, we will add `/openapi` (which returns list of supported versions) as a followup.